xml/cha_administration.xml

```diff
@@ -28,4 +28,187 @@
   </dm:docmanager>
  </info>
 
+ <sect1 xml:id="sec-admin-register">
+  <title>Registering instances</title>
+  <para>
+   Like other &suse; products, &sle; in public clouds has to be registered to
+   receive updates. There are different ways to register, depending on what
+   instance you register where.
+  </para>
+  <itemizedlist>
+   <listitem>
+    <para>
+     <literal>PAYG</literal> instances are registered against a local update
+     server, that means an update server in the cloud framework and region. This
+     is done automatically by the <systemitem
+      class="service">guestregister.service</systemitem> upon first boot.
+    </para>
+  </listitem>
+   <listitem>
+    <para>
+     <literal>BYOS</literal> instances have to be registered manually with your
+     &suse; registration key. You can register against a local update server,
+     the &scc;, or your own &susemgr; or &rmt; infrastructure.
+    </para>
+   </listitem>
+  </itemizedlist>
+  <para>
+   There are two different ways to register BYOS instances with the &scc;,
+   depending on the datestamp of the image the instance was created from:
+  </para>
+  <itemizedlist>
+   <listitem>
+    <para>
+     Images with a datestamp <emphasis>later</emphasis> than
+     <literal>20220103</literal> can be registered with
+     <command>registercloudguest</command>. This sets up
+     <command>&suseconnect;</command> to work against the cloud framework's update
+     infrastructure rather than the &scc;, resulting in significantly faster
+     package downloads. Refer to
+     <xref linkend="sec-admin-registercloudguest"/> for instructions.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Images with a datestamp <emphasis>prior</emphasis> to
+     <literal>20220103</literal> are registered with
+     <package>&suseconnect;</package>. Refer to <xref
+      linkend="sec-admin-suseconnect"/>. Alternatively, you can install
+     <command>registercloudguest</command> to benefit from faster downloads
+     from the local cloud update infrastructure.
```

Comment:
Depending on what we want to emphasis I think it would be advantages to turn this around and first speak of the requirement to install cloud-regionsrv-client, could read as follows:

"""
BYOS images with datestamp prior to 20220103 do not have the client side packages installed to register against the SUSE update images. If you want to register against the update infrastructure install

cloud-regionsrv-client, cloud-regionsrv-client-addon-azure,cloud-regionsrv-client-plugin-azure, regionServiceClientConfigAzure for Azure

cloud-regionsrv-client, cloud-regionsrv-client-plugin-ec2, regionServiceClientConfigEC2 for AWS

and

cloud-regionsrv-client, cloud-regionsrv-client-plugin-gce, regionServiceClientConfigGCE for GCE

to register against scc use SUSEConnect.
"""

In any event the install of "<command>registercloudguest</command>" is not sufficient as that would only bring the "cloud-regionsrv-client" package and that will not be enough as shown in the suggested change text.